Sources/MapboxSearch/PublicAPI/MapboxSearchVersion.swift

```diff
@@ -1,2 +1,2 @@
 /// Mapbox Search SDK version variable
-public let mapboxSearchSDKVersion = "1.0.0-rc.7"
+public let mapboxSearchSDKVersion = "1.0.0-rc.8"
```

Comment:
My only concern is this line. Point #6 [here](https://mapbox.slack.com/archives/C05FS658HAR/p1702662408729689?thread_ts=1702662189.709229&cid=C05FS658HAR) from @DzmitryFomchyn was suggesting something else. As discussed offline I know this was auto generated, but probably we will need to update it manually. What do you think @DzmitryFomchyn?

Reply:
This was resolved by SSDK-517 and merged in PR 142